boards/seeedstudio-gd32/include/periph_conf.h

```diff
@@ -35,6 +35,8 @@
 #define CONFIG_CLOCK_HXTAL      MHZ(8)      /**< HXTAL frequency */
 #endif
 
+#define SPI_DEV_1_USED              /**< Enable SPI_DEV(1) for the connected flash */
```

Comment:
When I hear connected SPI flash I see an opportunity to add an MTD device :D

Reply:
Of course, but we don't have a driver for SPI flash like the W25Q64 with the exception of ESP32.

Reply:
is it not just a standard SPI-NOR flash that's supported by `mtd_spi_nor`?

Reply:
Ah ok, `mtd_spi_nor` is already doing that job. I will try.

Reply:
> When I hear connected SPI flash I see an opportunity to add an MTD device :D

Too bad, this is the Seeedstudio GD32 RISC-V board, which I don't have and can't order anywhere at the moment. So I can't try it out.